content/docs/examples/multicluster/iks-icp/index.md

```diff
@@ -7,7 +7,7 @@ keywords: [kubernetes,multicluster,hybrid]
 
 This example demonstrates how to use Istio's multicluster feature to join 1
 [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster and 1 [IBM Cloud Kubernetes Service](https://console.bluemix.net/docs/containers/container_index.html) cluster together,
-using the [multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/).
+using the [VPN based multicluster installation instructions](/docs/setup/kubernetes/multicluster-install/vpn/).
```

Comment:
"flat-network multicluster installation instructions" for consistency

Reply:
I think it more advisable to consistently use "VPN-based" instead of "flat-network".